lib/ansible/modules/cloud/vmware/vmware_guest_vnc.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2018, Armin Ranjbar Daemi randjbar@gmail.com
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: vmware_guest_vnc
+
+short_description: Manages VNC remote display on virtual machines in vCenter
+
+description:
+  - Enables and disables VNC remote display on virtual machine.
+
+version_added: '2.6'
+
+author:
+  - Armin Ranjbar Daemi (@rmin) <randjbar@gmail.com>
+
+requirements:
+  - python >= 2.6
+  - PyVmomi
+
+options:
+  state:
+    description:
+      - Set the state of VNC on virtual machine.
+    choices:
+      - present
+      - absent
+    default: present
+    required: false
+  name:
+    description:
+      - Name of the virtual machine to work with.
+      - Virtual machine names in vCenter are not necessarily unique, which may be problematic, see C(name_match).
+    required: false
+  name_match:
+    description:
+      - If multiple virtual machines matching the name, use the first or last found.
+    default: first
+    choices:
+      - first
+      - last
+    required: false
+  uuid:
+    description:
+      - UUID of the instance to manage if known, this is VMware's unique identifier.
+      - This is required if name is not supplied.
```

Comment:
Add a `required_one_of` for name/uuid (the mutually exclusive is good, but this adds in module robustness as well)